docs/_sidebar.md

```diff
@@ -157,3 +153,5 @@
     - [7. Translations](/tutorial/twitter/tutorial-part-7.md)
     - [8. Unit Testing](/tutorial/twitter/tutorial-part-8.md)
     - [9. Virtual Lists](/tutorial/twitter/tutorial-part-9.md)
+  - [Qooxdoo Mobile App](/mobile/tutorial.md)
```

Comment:
All of the items above this new one are numbered. Should not this one be too?

Reply:
I think it's a different tutorial specifically for mobile apps - the above ones belong to the website version.